src/screens/Followings/index.js

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 import {ScrollView, StyleSheet} from 'react-native';
 import PropTypes from 'prop-types';
 import UsersFragment from '../DiscoveryScreenV2/fragment/UsersFragment';
-import {colors} from '../../utils/colors';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `colors` to `COLORS` is a good one as it follows the convention of using uppercase for constant values. This makes it clear that `COLORS` is a constant and should not be modified.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->